README.md

```diff
@@ -9,16 +9,9 @@ The MoveIt! Motion Planning Framework **for ROS 2.0**
 - [Get Involved](http://moveit.ros.org/documentation/contributing/)
 
 ## Milestones
-- [ ] Setup instructions
-  - [ ] Install instructions
-    - [ ] Ubuntu 18.04
-    - [ ] OS X 10.14
-- [ ] Update/setup infrastructure for development
-  - [x] Delete metapackages
-  - [x] Upgrade continuous integration for ROS 2.0
-  - [x] Refactor/cleanup folder hierarchy
+- [ ] Install instructions for Ubuntu 18.04 in README.md
```

Comment:
Suggesting to include Ubuntu and OS X given the fact that we're developing in both:
```suggestion
- [ ] Install instructions
  - [ ] Ubuntu 18.04
  - [ ] OS X 10.14
```